tailwind.config.js

```diff
@@ -6,7 +6,14 @@ module.exports = {
     extend: {
       gridTemplateColumns: {
         'chips-sheet': 'repeat(auto-fit, minmax(90px, min-content))',
-      }
+      },
+      width: {
+        '1/10': '10%',
+        '165': '165px',
+      },
```

Comment:
since tailwind config use only rems,  but we want to have exactly known about of px - we can use tailwind class like tw-w-[165px]